src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -22,18 +24,22 @@ public class AddCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
             + "Parameters: "
+            + PREFIX_STUDENTID + "ID "
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
+            + PREFIX_COURSE + "COURSE "
             + "[" + PREFIX_TAG + "TAG]...\n"
```

Comment:
```suggestion
            + PREFIX_TAG + "TAG\n"
```

I think since tag is now singular and compulsory, we should remove the square brackets indicating optional field and the ellipses indicating multiple tags are accepted.